performant-software/react-components
# Created Story and Docs for FileInputButton

### What this does

Closes #208 

### To Do

- [x] Create story
- [x] Add a label
- [x] Where do docs go?

### How to test
- Go to [the FileInputButton Story](https://performant-software.github.io/react-components/?path=/docs/components-semantic-ui-fileinputbutton--docs)
- Confirm there are docs
- Make sure you can follow and use them.
- Done! 💯
